Applications/ProductRatings/res/product_ratings.txt

```diff
@@ -0,0 +1,25 @@
+1 1 5
```

Comment:
rename this file to `sample_product_ratings.txt`?

Reply:
Done